extensions/python/src/main/java/com/hazelcast/jet/python/PythonService.java

```diff
@@ -69,7 +68,7 @@ final class PythonService {
         server = new JetToPythonServer(serviceContext.runtimeBaseDir(), logger);
         try {
             int serverPort = server.start();
-            chan = NettyChannelBuilder.forAddress("127.0.0.1", serverPort)
+            chan = serviceContext.channelFn().apply("127.0.0.1", serverPort)
                                       .usePlaintext()
```

Comment:
The `.usePlaintext()` is non configurable, e.g. if the user uses `.useTransportSecurity()` we override it by calling the `.usePlaintext()` here.

Also is there a case where the user would use something else then `NettyChannelBuilder.forAddress("127.0.0.1", serverPort)` to start from?

Maybe that can be the default and we just let the user modify it with the channel function of type `ConsumerEx<? extends ManagedChannelBuilder<?>>`

Or possibly we let the user start from scratch completely not passing the host and port at all, so it would be `SupplierEx<? extends ManagedChannelBuilder<?>>`

Reply:
My understanding was that since the connection is always to localhost, there's no point in encrypting it. The connection is always localhost, because that's where we configured the virtual python environment that we want to communicate to. And it's not possible to create a `ManagedChannelBuilder` without an host/port, or a target URI.

Reply:
Heh, you would be suprised what some customers may request :D Once I got a question if all data in the caches and traffic is encrypted in Hazelcast. IIRC we told them it's not necessary with the same reason... 

Do we even need the usePlaintext() here? If it works with both versions, I'd remove that line to not limit us if it's not necessary. However I think it's not a high prio, so you can create an issue to make proper tests for encrypted connection and to remove this line (IMHO it's fine as it is, may be improved after 5.2)

Reply:
TLS is the default. For it to work, we would have to configure certificates.
Let's stick to the original issue we're fixing: adding the possibility to configure maximum request size, this is another problem.